pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -787,6 +787,18 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			}
 		}
 
+		// Set VM CPU features
+		if vmi.Spec.Domain.CPU.Features != nil {
+			if len(vmi.Spec.Domain.CPU.Features) > 0 {
```

Comment:
you don't have to have this check here. If vmi.Spec.Domain.CPU.Features has 0 length, then for loop won't start.